src/time_man.cpp

```diff
@@ -278,4 +280,5 @@ void TimeManager::serialise(JsonDocument &doc)
   doc["time"] = time;
   doc["local_time"] = local_time;
   doc["offset"] = offset;
+  doc["uptime"] = millis() / 1000 - bootTimeSecs; // Note: overflow not checked because this gives about 136 years (2^32/60/60/24/365)
```

Comment:
Not sure if this is the best place, but `/status` is also maybe not the best place either. Lets leave here and see how people find it. Don't think you really need the `- bootTimeSecs`, `millis()` returns the time since boot anyway, and also not sure your comment about overflow is the correct time (https://www.arduino.cc/reference/en/language/functions/time/millis/):

> Returns the number of milliseconds passed since the Arduino board began running the current program. This number will overflow (go back to zero), after approximately 50 days.

Reply:
Oh! My bad! I have a Java developer background and I thought this function would return the Unix time (traditional time-stamp). So now I understand why all implementations have a rollover at 50 days! I will change. This is not correct indeed.

Reply:
I've change to a more flexible implementation on 64 bits which is working fine as long it is called once every 47 days